wallet/src/Pos/Wallet/Web/Pending/Submission.hs

```diff
@@ -110,6 +112,10 @@ submitAndSavePtx
     :: MonadWalletWebMode m
     => PtxSubmissionHandlers m -> EnqueueMsg m -> PendingTx -> m ()
 submitAndSavePtx PtxSubmissionHandlers{..} enqueue ptx@PendingTx{..} = do
+    -- this should've been checked before, but just in case
+    when walletTxCreationDisabled $
+        throwM $ InternalError "Transaction creation is disabled by configuration!"
```

Comment:
This gets called by `resubmitTx` from the re-submission worker (and it handles and ignores exceptions), so this is ok to stop re-submitting transactions.

Reply:
This function is used in all wallet methods which deal with transaction submission, including `newPayment`, so I added this check here as extra measure for possible new transaction-submission-related endpoints